tests/execution.rs

```diff
@@ -4137,3 +4126,35 @@ fn test_invalid_exit_or_return() {
         );
     }
 }
+
+#[test]
+fn test_unregistered_syscall() {
```

Comment:
This is essentially an unreachable condition.

Reply:
Remove it then? I usually test all code paths.

Reply:
Removed.